components/eam/src/chemistry/yaml/mo_photo/f90_yaml/cloud_mod_end_yml.f90

```diff
@@ -0,0 +1,38 @@
+#ifdef YAML_PHOTO
+  if(yaml%flag_print) then ! if this column exists in lchnk
+
+     !write output header
+     call write_output_header(unit_output)
+
+     !start writing data
+     
+        call write_var(unit_output,'eff_alb',eff_alb(yaml%lev_print))
+        call write_var(unit_output,'cld_mult',cld_mult(yaml%lev_print))
+
+     !writes aerosol mmr from state%q or q vector(cloud borne and interstitial) in the output python module
+     !"aer_num_only" is .ture. if printing aerosol num only
+     !call write_output_aerosol_mmr_from_stateq(unit_output, fld_name, field, aer_num_only, inp_out_str)
+
+     !close the output file
+     close(unit_output)
+     call freeunit(unit_output)
+  endif
+
+!------------------------------------------------------------------------------------------------------------------------------
+! This part of the code is for finding the best lat/lon and n_calls.
+
```

Comment:
Should the following section be commented out to prevent side effect issues with the printouts?